projects/packages/sync/tests/php/test-actions.php

```diff
@@ -0,0 +1,95 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
I'm not sure if I should have just added these tests to the existing Sync unit test files in the `plugins/jetpack/tests/php/sync` directory.

Reply:
Here is more appropriate, the existing tests in the jetpack folder eventually need to be split having unit tests in the package then only the integration level tests in jetpack.